nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/ElasticSearchRestProcessor.java

```diff
@@ -75,6 +76,19 @@
             .required(true)
             .build();
 
+    Relationship REL_FAILURE = new Relationship.Builder()
+            .name("failure")
+            .description("All flowfiles that fail for reasons unrelated to server availability go to this relationship.")
+            .build();
+    Relationship REL_RETRY = new Relationship.Builder()
+            .name("retry")
+            .description("All flowfiles that fail due to server/cluster availability go to this relationship.")
+            .build();
+    Relationship REL_SUCCESS = new Relationship.Builder()
+            .name("success")
+            .description("All flowfiles that succeed in being indexed into ElasticSearch go here.")
```

Comment:
Maybe "transmitted to ElasticSearch"? Unless "indexed" is a generic enough term that users would know covers upserts, deletes, and any other supported operation.